addon/session-stores/cookie.js

```diff
@@ -37,18 +37,21 @@ const persistingProperty = function(beforeSet = function() {}) {
 
   ```js
   // app/controllers/login.js
-  export default Ember.Controller.extend({
-    rememberMe: computed({
-      get(key) {
-        return false;
-      },
-      set(key, value) {
-        let expirationTime = value ? (14 * 24 * 60 * 60) : null;
-        this.set('session.store.cookieExpirationTime', expirationTime);
-        return value;
-      }
-    })
-  });
+  import Controller from '@ember/controller';
+
+  export default class LoginController extends Controller {
+    _rememberMe = false;
+
+    get rememberMe() {
+      return this._rememberMe;
+    }
+
+    set rememberMe(value) {
+      let expirationTime = value ? (14 * 24 * 60 * 60) : null;
+      this.set('session.store.cookieExpirationTime', expirationTime);
```

Comment:
The `session` service injection is missing.

Reply:
added